src/opi/external_methods/process.py

```
@@ -0,0 +1,78 @@
+import subprocess
+
+
+class Process:
+    """
+    Class that runs a process, monitors it and terminates it.
+    """
+
+    def __init__(self):
+        """
+        Init Process object
+        """
+        # Variable for storing the process
+        self.process = None
+
+    def start(self, cmd: list[str], pipe: bool=False) -> None:
+        """
+        Starts a subprocess
+
+        Attributes
+        ----------
+        cmd: str
+            List of arguments to be executed
+        pipe: bool, default: False
+            Determines whether to hold the channel open for piping input in or not.
+
+        Raises
```

The purpose of the "Raises" block is to state which exceptions a function might throw when called. But as you catch all and handle all of these exceptions already, none of these will ever be raised by this function.
I would suggest not catching them and let them propagate naturally.